### PR TITLE
Clarify TOS copy in signup

### LINKED ISSUE #33289: Terms of service copy on signup form could be improved.
## URL
https://wordpress.com/start/user

## Actual

The terms of service copy reads:

> By creating an account via any of the options **below**, you agree to our Terms of Service.

The use of the word **below** suggests that if you sign up via the form above that the TOS don't apply to you.

## Expected

We should probably rewrite this to include all signup options.

### PR DESCRIPTION
**Pull request by alexislloyd**
## Changes proposed in this Pull Request
Updated copy to clarify that *any* account creation method is subject to the TOS.

### Before
before

### After
after

### Testing instructions

* Go to /start/user **while logged out** to view.

Fixes #33289

### CODE REVIEW

File: client/blocks/signup-form/index.jsx [hunk 1]
```diff
@@ -611,7 +611,7 @@ class SignupForm extends Component {
 			return (
 				<p className="signup-form__terms-of-service-link">
 					{ this.props.translate(
-						'By creating an account via any of the options below, you agree to our {{a}}Terms of Service{{/a}}.',
+						'By creating an account, you agree to our {{a}}Terms of Service{{/a}}.',
```
Comment (a8ci18n) [inline client/blocks/signup-form/index.jsx]: Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'By creating an account you agree to our {{a}}Terms of Service{{/a}}.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by scruffian** [comment]
This change looks great. For any changes to this copy we have to loop in legal. cc @jennyzhu.

**Comment by dreiris** [comment]
I agreed! The changes look good :)

**Comment by alexislloyd** [comment]
I just updated this as per a Slack conversation with @jennyzhu to make sure the copy can be both clear to users and comply with our legal requirements. @davemart-in let me know if you have any comments before I ship it!

File: client/blocks/signup-form/index.jsx (hunk 1 above)
Comment (scruffian) [reply client/blocks/signup-form/index.jsx]: I guess we should remove the `,`...

**Comment by davemart-in** [comment]
@alexislloyd Looks good!

**Comment by jennyzhu** [comment]
Just a heads up that when going through the Happy.Tools sign up flow, I found an older version of this page ([URL](https://wordpress.com/log-in?client_id=46367&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D46367%26redirect_uri%3Dhttps%253A%252F%252Fschedule.happy.tools%252Fregister%252Fsignup%26response_type%3Dtoken%26scope%3Dauth%26state%3Dwpcom-register)). How can we make sure to keep all versions of these pages consistent and up to date?

**Comment by davemart-in** [comment]
@jennyzhu the changes in this specific PR were made in the [WP.com **signup** form](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/signup-form/index.jsx).

The screen you've highlighted is the [WP.com **login** form](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/login/login-form.jsx), which is separate.

While looking at this, I also noticed that the [CrowdSignal signup form](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/signup-form/crowdsignal.jsx#L144) which is separate from the WP.com signup form could also be updated.

These would need to take place in a separate PR (since this PR has already been merged).

cc'ing @sixhours to see if these would be a good fit for #fixtheflows, else I'm happy to own this.

**Comment by sixhours** [comment]
@davemart-in That's right up our alley!

**Pull request merged**